### ISSUE TITLE
lux serve does not work

### ISSUE THREAD

**Issue by such** [postlight/lux#352]
Hello,

I tried:

```
$ lux new test-lux
$ lux generate model users
$ lux serve
```

It seems stuck (No more message after the building phase). Tested on `master` as well with no better luck.
It was working before I generated the user model. It is still not working after I've executed the user migration.

**Comment by zacharygolba** [comment]
Hey, thanks for using Lux! 😃 

The reason `lux serve` is not working is because you have pending migrations that need to run. To migrate your db execute `lux db:migrate` from the root of your project.

#316 implemented a fix that prevents these types of errors from failing silently but it has not been released yet.

**Comment by such** [comment]
Hey,
Thanks for building this exciting project.
As I said though it still does not work after `lux db:migrate`...

**Comment by zacharygolba** [comment]
Ah I forgot to mention `lux db:create`!

So the workflow should look like this:

```bash
lux new test-lux
cd test-lux
lux db:create
lux generate model user
lux db:migrate
lux serve
```

If that doesn't work, would you mind providing some information about the environment your running in (i.e operating system, lux version, node version, etc)?

**Comment by seawatts** [comment]
I am still getting this same issue.

**Comment by seawatts** [comment]
Got this working by using master and also running 
``` 
lux g serializer user
lux g controller user
```

**Comment by zacharygolba** [comment]
Thank you @such and @seawatts for helping out with this issue! I'm going to go ahead and close it now because this is resolved in the latest release (`1.0.0-rc.7`).

![omfhp8g6ke](https://cloud.githubusercontent.com/assets/7378772/18236772/d76d7822-72f7-11e6-9f2c-c57676e3e158.gif)
